plugins/Marketplace/vue/src/PluginList/CTAContainer.vue

```diff
@@ -108,10 +112,12 @@
     </a>
   </template>
 
-  <MoreDetailsButton
-    v-else
-    :plugin-name="plugin.name"
-  />
+  <template v-else>
+    <MoreDetailsButton
+      v-if="!inModal"
```

Comment:
We probably should add more of those `!inModal` checks. For example the Help link is opening the old modal above the new modal when clicked :see_no_evil:

Reply:
Good catch! With the new modal, the help link doesn't even need to exist as a separate component and can just open the details modal as the whole card does. Will adjust the behaviour.

Reply:
I've refactored the HelpLink and MoreDetailsButton components into one as the mechanism is identical and they now do the same thing just looking differently. I've added the `inModal` condition to multiple more places and now it doesn't show inside the modal any links or buttons that would open the (old) modal again.